app/javascript/components/admin/AdminTable.js

```diff
@@ -525,12 +527,14 @@ class AdminTable extends React.Component {
         // This has been addressed and recently merged: https://github.com/react-csv/react-csv/pull/193
         // Once this gets released and we update our version this code won't be necessary.
         const csvData = response.data.user_rows.map(userData => {
+          // eslint-disable-next-line no-unused-vars
+          const { active_state, lock_reason, auto_lock_reason, ...dataToExport } = userData;
```

Comment:
For each user, we might need access to those three pieces of data in the User Edit Modal:

**active_state:** are they active/inactive? 
**lock_reason:** why is the user locked? Options are: "No longer an employee", "No longer needs access", "Other", "Automatically Locked by the System", BLANK
**auto_lock_reason:** Why is the user automatically locked by the system? This text is shown in the modal in red with an exclamation point for users that are Automatically Locked by the System (e.g., Temporary password expired)

**So for a locked user...**
When we open the Edit Modal, we show their lock_reason and/or their auto_lock_reason (if applicable)
If a locked user is then unlocked in the Edit Modal, we immediately update the Status to their active_state and disable the dropdown 

We don't want to export these values to CSV because they are only really useful in the context of the User Edit modal

Reply:
Sarah had a similar comment, but it's hidden now because it was resolved so you wouldn't have seen it! Here is the link: [discussion_r780553526](https://github.com/SaraAlert/SaraAlert/pull/1180#discussion_r780553526)

Let me know what you think!

Reply:
got it, can you just add a comment explaining that in the code as well? thanks!

Reply:
Sure! Let me know if that comment makes sense to you: 
[0cc8b92251b8bc8ac8e5d0466745182ffbb5e21d](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/0cc8b92251b8bc8ac8e5d0466745182ffbb5e21d)

Reply:
awesome, thanks!